org.elysium.parent/org.elysium/src/org/elysium/importuri/LilyPondImportUriResolver.java

```diff
@@ -32,6 +32,15 @@
 	@Inject
 	private ILilyPondPathProvider lilyPondPathProvider;
 
+	public static boolean isAbsolute(String uriString) {
+		try {
+			return new File(uriString).isAbsolute();
+		}catch(Exception e) {
+			//ignore for now, spaces in file names cause problems
```

Comment:
Exactly what kind of problems do spaces in file names cause?

Reply:
At least on Windows java.net.URI.create... with filenames containing white spaces caused an exception. It is possible that by using new File(name) this type of exception will not occur.

However, as this is an intermediate step and not the full include-handling-refactoring and the isAbsolute-check is only used for validation and the is-absolute-hint for refactorings, I considered ignoring possible Expeptions OK for now.

Actually, the newly introduced test for the isAbsolute-method could be extended to cases where file or folder names contain whitespaces...